front/app/components/admin/PostManager/components/FilterSidebar/FilterRadioButton/index.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+import { Box } from '@citizenlab/cl2-component-library';
+import { hideVisually } from 'polished';
+import styled from 'styled-components';
+
+const HiddenRadio = styled.input.attrs({ type: 'radio' })`
```

Comment:
[Ticket created](https://www.notion.so/govocal/Radio-without-checkbox-1629663b7b26807ea85ada67a1ac2fb4?pvs=4) to include this functionality in our component library.